docs/_docs/templates.md

```diff
@@ -105,6 +105,35 @@ you come up with your own tags via plugins.
         </p>
       </td>
     </tr>
+    <tr>
+      <td>
+        <p class="name"><strong>Date to String in ordinal US style</strong></p>
+        <p>Format a date to ordinal, UK, short format.</p>
+      </td>
+      <td class="align-center">
```

Comment:
Please **reorder** the table rows here such that information flows gracefully. For example:

Filter Desc. | Example
--- | --- 
**Date to String** | `date_to_string`
**Date to String in UK Ordinal** | `date_to_string: "ordinal"`
**Date to String in US Ordinal** | `date_to_string: "ordinal", "US"`
**Date to Long String** | `date_to_long_string`
**Date to Long String in UK Ordinal** | `date_to_long_string: "ordinal"`
**Date to Long String in US Ordinal** | `date_to_long_string: "ordinal", "US"`